tests/DataObjectTest.py

```diff
@@ -110,3 +110,28 @@ def __init__(self, a=None, **kw):
     def test_rdfs_comment_property(self):
         a = DataObject(rdfs_comment='Hello')
         self.assertIn('Hello', a.rdfs_comment())
+
+    def test_context_getter(self):
+        a = DataObject(rdfs_comment='Hello')
+        self.assertIsNone(a.context)
+
+    def test_context_setter(self):
+        a = DataObject(rdfs_comment='Hello')
+        a.context = 42
+        self.assertEquals(a.context, 42)
+
+    # TODO: only test when raise here
+    def test_datatype_property(self):
+        print(DataObject(rdfs_comment='Hello').DatatypeProperty(dict()))
```

Comment:
Prefer to use self.assertRaises, especially if you have a specific exception in mind.

Reply:
Oh sorry, I had to go and I push during debugging. But it's ok for self.assertRaises, thanks.